src/catkin_pkg/changelog_generator.py

```diff
@@ -94,7 +97,11 @@ def _get_version_tags(vcs_client):
 def _get_latest_version_tag_name(vcs_client):
     # get latest tag
     tag_name = vcs_client.get_latest_tag_name()
-    version_tag_name = tag_name if re.match(r'^\d+\.\d+.\d+$', tag_name) else None
+    version_tag_name = None
+    if re.match(r'^\d+\.\d+.\d+$', tag_name):
+        version_tag_name = tag_name
+    else:
+        raise RuntimeError("Tag name '{}' failing to meet criteria.".format(tag_name))
```

Comment:
This might help users to narrow down where is the root cause is, but might not be helpful enough to come with a fix because what is the "criteria" isn't mentioned. What can we tell users here?